### PR TITLE
README.md: Document expected use of lowerCamelCase

### LINKED ISSUE #1008: documentation: README.md - Document lowerCamelCase keys
In https://github.com/exercism/problem-specifications/issues/987 it was decided that we should be consistent and use `lowerCamelCase` for all json key names. This needs to be documented in the [README.md test data format section](https://github.com/exercism/problem-specifications/blob/master/README.md#test-data-format-canonical-datajson)

It's currently mentioned in the comments within the example json, but I think it's worth explicitly mentioning outside of the example text that all keys should use that format.

**Task**

Update the "There are also some convention about expected that you must follow:" section of [README.md](https://github.com/exercism/problem-specifications/blob/master/README.md) to state that all json keys should be `lowerCamelCase`

### PR DESCRIPTION
**Pull request by rpottsoh**
closes #1008

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,7 @@ example, what kinds of implementation mistakes might this case help us find?
 
 There are also some convention about `expected` that you must follow:
 
+  - all keys should follow [lowerCamelCase](http://wiki.c2.com/?LowerCamelCase) convention.
```
Comment (Insti) [inline README.md]: Looks good. 👍 

Minor grammar issues:
**A**ll keys should follow **the** [lowerCamelCase](http://wiki.c2.com/?LowerCamelCase) convention.

The following `if`s lacking capital `I`s is a separate unrelated grammatical issue that this PR need not address.

**Comment by rpottsoh** [comment]
Very good.  I'll capitalize the sentence as you suggest and leave the others as is, for a future PR to deal with.

Thanks


Sent from my android device.

File: README.md (hunk 1 above)
Comment (rpottsoh) [reply README.md]: Changes made as requested.

**Comment by Insti** [comment]
Thanks @rpottsoh :heart:

**Pull request merged**